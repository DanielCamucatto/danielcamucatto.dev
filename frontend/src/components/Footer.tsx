
export default function Footer() {
  return (
    <footer className="max-w-md pb-16 text-sm text-slate-500 sm:pb-0">
      <p>
        Constru√≠do com{" "}
        <a
          href="https://reactjs.org/"
          className="font-medium text-slate-400 hover:text-teal-300 focus-visible:text-teal-300"
          target="_blank"
          rel="noreferrer noopener"
          aria-label="React (opens in a new tab)"
        >
          React
        </a>{" "}
        e{" "}
        <a
          href="https://tailwindcss.com/"
          className="font-medium text-slate-400 hover:text-teal-300 focus-visible:text-teal-300"
          target="_blank"
          rel="noreferrer noopener"
          aria-label="Tailwind CSS (opens in a new tab)"
        >
          Tailwind CSS
        </a>
        .
      </p>
    </footer>
  );
}