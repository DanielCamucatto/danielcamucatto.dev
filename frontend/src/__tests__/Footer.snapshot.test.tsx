import { render } from '@testing-library/react';
import Footer from '../components/Footer';
describe('Footer snapshot', () => {
  it('deve bater o snapshot', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
