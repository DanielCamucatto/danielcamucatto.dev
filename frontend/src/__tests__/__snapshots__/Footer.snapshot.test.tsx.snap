// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Footer snapshot deve bater o snapshot 1`] = `
<DocumentFragment>
  <footer
    class="max-w-md pb-16 text-sm text-slate-500 sm:pb-0"
  >
    <p>
      Constru√≠do com 
      <a
        aria-label="React (opens in a new tab)"
        class="font-medium text-slate-400 hover:text-teal-300 focus-visible:text-teal-300"
        href="https://reactjs.org/"
        rel="noreferrer noopener"
        target="_blank"
      >
        React
      </a>
       e 
      <a
        aria-label="Tailwind CSS (opens in a new tab)"
        class="font-medium text-slate-400 hover:text-teal-300 focus-visible:text-teal-300"
        href="https://tailwindcss.com/"
        rel="noreferrer noopener"
        target="_blank"
      >
        Tailwind CSS
      </a>
      .
    </p>
  </footer>
</DocumentFragment>
`;
